patches/blockprotocol+0.0.1.patch

```diff
@@ -52,12 +76,17 @@ index 205630a..ccd3499 100644
  export type BlockProtocolCreateLinksAction = {
    sourceAccountId?: string | null;
    sourceEntityId: string;
-@@ -163,15 +185,33 @@ export type BlockProtocolCreateLinksAction = {
++  sourceEntityTypeId?: string;
++  sourceEntityTypeVersionId?: string;
+   destinationAccountId?: string | null;
+   destinationEntityId: string;
+   destinationEntityVersionId?: string | null;
+@@ -163,15 +193,33 @@ export type BlockProtocolCreateLinksAction = {
    path: string;
  };
  
 +export type BlockProtocolUpdateLinksAction = {
-+  sourceAccountId: string;
++  sourceAccountId?: string;
 +  sourceEntityId: string;
 +  path: string;
 +  updatedOperation: BlockProtocolLinkedAggregationOperationInput;
```

Comment:
It looks like there are two of these now?

Reply:
Deduped this as well.